packages/rspack/package.json

```diff
@@ -35,6 +35,7 @@
     "@rspack/plugin-node-polyfill": "workspace:^",
     "@types/watchpack": "^2.4.0",
     "@types/webpack": "5.28.3",
+    "@types/webpack-dev-server": "^4.7.2",
```

Comment:
why `@types/webpack-dev-server` is needed here?

Reply:
see https://github.com/web-infra-dev/rspack/blob/3c082533ac76e1f2c2a0e0c5ed7955545941a87a/packages/rspack/src/config/zod.ts#L5